qemu/tests/cfg/pci_bridge.cfg

```diff
@@ -1,19 +1,46 @@
 - pci_bridge: install setup image_copy unattended_install.cdrom
     virt_test_type = qemu
     login_timeout = 360
+    type = pci_bridge
+    no q35
```

Comment:
When only use `no q35` here, it means `i440fx`, `pseries`, `arm64-mmio`, `arm64-pci`, `riscv64-mmio`,  `s390-virtio`.
As we know, `i440fx` and `pseries` support pci-bridge.
I am not very sure about others, but s390x don't support pci-bridge.

Reply:
Updated, Thanks!